SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -9,8 +9,9 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialNetBitReader);
 
-FSpatialNetBitReader::FSpatialNetBitReader(USpatialPackageMapClient* InPackageMap, uint8* Source, int64 CountBits, TSet<FUnrealObjectRef>& InUnresolvedRefs)
+FSpatialNetBitReader::FSpatialNetBitReader(USpatialPackageMapClient* InPackageMap, uint8* Source, int64 CountBits, TSet<FUnrealObjectRef>* InDynamicRefs, TSet<FUnrealObjectRef>& InUnresolvedRefs)
```

Comment:
Why make InDynamicRefs a pointer? Seems more straight forward to just require it.